docs/README.md

```diff
@@ -1,40 +1,30 @@
 # Macaron documentation
-This directory contains the Sphinx documentation for Macaron.
+This directory contains the source for the documentation of Macaron hosted [here](https://oracle-samples.github.io/macaron/).
 
-## Generating the HTML documentation from source
-**Note**: We only include the API documentation at the moment. Other sections in the documentation will be migrated in the future.
-
-First, make sure you have all the dependencies required to build the documentation (Assume we run the following commands in the root directory of this repo).
-
-**TODO**: Add docs dependencies to `pyproject.toml`.
+## Build the documentation
 
+To build the documentation, please follow these steps:
+1. Setup the dev environment for Macaron using the instructions [here](../README.md#getting-started).
+2. Build the documentation by running this command from the root directory of Macaron:
 ```
-python -m pip install --editable .
+make docs
 ```
 
-To generate documentation in html form:
-
-```
-sphinx-build -b html docs/source docs/build -E
-```
+This command will build and generate the documentation into `docs/_build/html`. To view it locally, run (with the dev environment activated):
 
-Or within the [docs](./) folder, run:
 ```
-make html
+python3 -m http.server -d docs/_build/html
 ```
 
-To view the html documentation on localhost:
+## Developer guide
 
-```
-python3 -m http.server -d docs/_build/html
-```
+### The API reference.
 
-## For developers: Generating the API docs.
-The Sphinx API docs for Macaron can be generated using the [sphinx-apidoc](https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html) tool that comes with `sphinx`.
+The API reference is generated using the [sphinx-apidoc](https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html) tool.
 
-From within the root directory of Macaron, run:
+From within the root directory of Macaron, run (with the dev environment activated):
 ```
 sphinx-apidoc --no-toc --module-first --force --maxdepth 1 --output-dir docs/source/pages/apidoc/ src/
 ```
 
-The output RST files will be generated into [pages/apidoc/](./source/pages/apidoc/) where it will be picked up when we build the documentation.
+The API reference RST files will be generated into `docs/source/pages/apidoc/` where it will be picked up when we build the documentation using the instructions [above](#build-the-sphinx-documentation).
```

Comment:
```suggestion
This command will  generate the API reference RST files into `docs/source/pages/apidoc/`. Make sure to check in the changed source files to the repository.
```

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/276/commits/4937b2d0f10affbec842dfa78e23213ec6a6a1d1